### ISSUE TITLE
ESM support

### ISSUE THREAD

**Issue by literat** [lirantal/lockfile-lint#207]
**Is your feature request related to a problem? Please describe.**

I would like to use file-based configuration in ESM syntax; however, even the `cosmiconfig` supports the syntax, I was not able to make it work.

**Describe the solution you'd like**

I would like to ESM syntax to be supported:

```js
export default {
  allowedHosts: ['npm'],
  path: 'yarn.lock',
  type: 'yarn',
  validatePackageNames: true
}
```

**Describe alternatives you've considered**

I have tried the config names suggested by your [documentation](https://github.com/lirantal/lockfile-lint/tree/main/packages/lockfile-lint#file-based-configuration), and I have also tried to use a `.mjs` extension that should work based on [cosmiconfig documentation](https://github.com/cosmiconfig/cosmiconfig). Also tried the default and named exports. None of them works.

Based on the code in this repository, I have configured my project similarly using CJS syntax. However, I am trying to prefer the ESM. It will be nice if this package works with ESM as well.

Thanks for this package and your open source work.

**Comment by lirantal** [comment]
@literat this is now published new major version 5 of the CLI

**Comment by literat** [comment]
@lirantal Working. Thanks 👍
